Mathlib/LinearAlgebra/Matrix/HermitianFunctionalCalculus.lean

```diff
@@ -0,0 +1,162 @@
+/-
+Copyright (c) 2024 Jon Bannon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jon Bannon, Jireh Loreaux
+-/
+
+import Mathlib.LinearAlgebra.Matrix.Spectrum
+import Mathlib.LinearAlgebra.Eigenspace.Matrix
+import Mathlib.Analysis.NormedSpace.Star.ContinuousFunctionalCalculus
+import Mathlib.Topology.ContinuousFunction.UniqueCFC
+import Mathlib.Analysis.NormedSpace.Star.Matrix
+import Mathlib.Algebra.Star.Unitary
```

Comment:
Surely not all of these imports are necessary?

Reply:
You are right! I winnowed this down to four.